packages/overlay/src/vaadin-overlay.d.ts

```diff
@@ -3,10 +3,10 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { ControllerMixin } from '@vaadin/component-base/src/controller-mixin.js';
```

Comment:
This import was unused, so I removed it. Apparently our ESLint config does not catch these.